src/App/App.js

```diff
@@ -64,14 +64,14 @@ class App extends Component {
 					<h1>Rancid Tomatillos</h1>
 				
 					<nav>
-						<NavLink to='/movies'>
+						<NavLink to='/'>
```

Comment:
Small thing - I think we should add the onClick to our NavLink, rather than keep it on the section.